streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -232,8 +325,19 @@ private void runJoinBenchmark(final KafkaStreams streams, final String nameOfBen
     }
 
 
+    public void processStream(String topic) throws Exception {
+        // see if we're in the load phase
+        if (loadPhase) {
+            // if we run all tests, the produce test will have already loaded the data
+            if (testName.equals(ALL_TESTS)) {
+                // Skipping loading phase since previously loaded
+                return;
+            }
+            System.out.println("processStream loading phase on topic: " + topic);
+            produce(topic, VALUE_SIZE, "simple-benchmark-process-stream-load", numRecords, true, numRecords, false);
+            return;
+        }
```

Comment:
Why this and other 4 tests below cannot reuse the `runGenericBenchmark` common function? It seems it is not necessary to use a separate thread for these tests to distinguish them from others that can leverage this common function.